package.json

```diff
@@ -4,27 +4,39 @@
   "private": true,
   "dependencies": {
     "bootstrap": "3",
+    "diff2html": "^2.3.3",
+    "less": "^2.7.3",
```

Comment:
bootstrap

Reply:
How come?
I understood we're using Bootstrap through compiled `bootstrap/dist/css/bootstrap.css` and `react-bootstrap` so it should not collide with our preprocessors, does it?

I also tried to use Sass in this project, and it seems to work as you can see at https://github.com/dpordomingo/code-annotation/pull/1/files

Reply:
please read code. I import bootstrap variables. Later I may also need to import bootstrap functions. The agreement was to use the same preprocessor as a component library does. Because bootstrap4 isn't ready to use with react yet, we use less.